packages/expo-ui/ios/ProgressView.swift

```diff
@@ -27,4 +28,13 @@ struct ProgressView: ExpoSwiftUI.View {
         $0.progressViewStyle(.linear)
       }
   }
+  
+  @ViewBuilder
+  private var progressView: some View {
+    if let timerInterval = props.timerInterval, timerInterval.count >= 2, let lower = timerInterval.first, let upper = timerInterval.last, lower <= upper, #available(iOS 16.0, *) {
```

Comment:
I think it'll look cleaner this way
```suggestion
    if let timerInterval = props.timerInterval, timerInterval.count >= 2, 
      let lower = timerInterval.first, 
      let upper = timerInterval.last, lower <= upper, 
      #available(iOS 16.0, *) {
```